tests/is-winner/pass_afonso.js

```diff
@@ -0,0 +1,42 @@
+const isWinner = (candidate) =>
+  winners()
+    .getWinner(candidate)
+
+    .catch((err) => Promise.reject(candidate + ' never was a winner'))
+
+    .then((countryInfo) =>
```

Comment:
this chaining do not benefit from `async` so it would be better too inline it in the next one.